index/index.md

```diff
@@ -57,8 +57,8 @@ The **Machine Translate Foundation** is building **open information** and **comm
 The content covers everything about machine translation, from products to research, and from history to news.
 
 > #### Featured events
-> - [AMTA 2024](/amta2024) - 30 September-02 October, Chicago 🇺🇸
-> - [EAMT 2024](/eamt2024) - 24-27 June, Sheffield 🇬🇧
+> - [AMTA 2024](/amta-2024) - 30 September-02 October, Chicago 🇺🇸
```

Comment:
But amta2023 we still have without dash.  Probably should all be with dash, to represent the space.